app/views/signInMobile.scala.html

```diff
@@ -0,0 +1,52 @@
+@import models.user.User
+@(signInForm: Form[com.mohiva.play.silhouette.api.util.Credentials], url: String = "/")(implicit env: com.mohiva.play.silhouette.api.Environment[User, _], request: RequestHeader, lang: Lang)
+
+@import views.html.bootstrap._
+
+@main("Project Sidewalk - Sign In", Some("/signInMobile")) {
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0">
+    </head>
+
+    @request.flash.get("success").map { msg =>
+        <div class="col-md-6 col-md-offset-3 alert alert-success"
+        style="float:none;margin-top:20px;margin-bottom:0px">
```

Comment:
Can you move all of the style stuff out of here and into CSS? I created a new `mobileAuthenticate.css` file that you can add stuff into if it is most appropriate in there.

Reply:
I don't see any `mobileAuthenticate.css` file i'm not sure if that's on my end or not. I don't think it is as I did git pull

Reply:
It's in `public/stylesheets/mobileAuthenticate.css`. Also just to make sure, are you using the `1904-Mobile-Sign-In-Page` branch? And not the old `1904-Mobile-Sign-In` or `1904-Sign-In-Mobile` branches? I have since deleted those old branches from Github because I accidentally checked out the wrong one once and got confused. But you might still have it on your local machine. If you were accidentally on the wrong branch, that would also explain the .ivy2 issues you were having.

Reply:
Fixed all occurrences of this issue